L1Trigger/L1TMuonEndCap/plugins/L1TMuonEndCapTrackProducer.cc

```diff
@@ -26,22 +26,14 @@ void L1TMuonEndCapTrackProducer::produce(edm::Event& iEvent, const edm::EventSet
   track_finder_->process(iEvent, iSetup, *out_hits, *out_tracks);
 
   // Convert into uGMT format
-  uGMT_converter_->convert_all(*out_tracks, *out_cands);
+  uGMT_converter_->convert_all(iEvent, *out_tracks, *out_cands);
 
   // Fill the output products
   iEvent.put(std::move(out_hits)  , "");
   iEvent.put(std::move(out_tracks), "");
   iEvent.put(std::move(out_cands) , "EMTF");
 }
 
-void L1TMuonEndCapTrackProducer::beginJob() {
```

Comment:
Thought this was kind of standard ... do we want to get rid of them?  Maybe @rekovic has an opinion.